test/test_memory_layout.cc

```diff
@@ -111,7 +111,7 @@ BOOST_AUTO_TEST_CASE(test_layout_arrays)
         Arrays arrays_v;
         size_t expected[] = {
             MemLayout::FLAG_MEMCPY,
-               reinterpret_cast<uint8_t*>(&arrays_v.a_v_numeric) - reinterpret_cast<uint8_t*>(&arrays_v),
+               static_cast<ptrdiff_t>(reinterpret_cast<uint8_t*>(&arrays_v.a_v_numeric) - reinterpret_cast<uint8_t*>(&arrays_v)),
```

Comment:
Well, now I'm telling you to get a Prozac ;-)

No problem, I enjoy getting proper proofreads: This is an honour, not a burden.